akka-http-core/src/main/scala/akka/http/impl/engine/server/HttpServerBluePrint.scala

```diff
@@ -154,10 +155,16 @@ private[http] object HttpServerBluePrint {
 
     val responseRendererFactory = new HttpResponseRendererFactory(serverHeader, responseHeaderSizeHint, log)
 
+    val errorHandler: Throwable ⇒ Unit = {
+      // socket timeouts should not result in errors in the log. See 19058.
+      case timeout: HttpConnectionTimeoutException ⇒ log.debug(s"Closing HttpConnection due to timeout: ${timeout.getMessage}")
+      case t                                       ⇒ log.error(t, "Outgoing response stream error")
```

Comment:
I just wanted to keep the current behaviour (see `errorLogger(log, "Outgoing response stream error")).named("errorLogger"))`)